job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/WeblogicDeployerContext.groovy

```diff
@@ -0,0 +1,129 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.Context
+import javaposse.jobdsl.dsl.ContextHelper
+import org.apache.commons.lang.RandomStringUtils
+
+/**
+ * DSL Support for the weblogic-deployment-plugin.
+ */
+class WeblogicDeployerContext implements Context {
+
+    /**
+     * Enumeration of available deployment stage modes.
+     */
+    static enum WeblogicDeploymentStageModes {
+
+        BY_DEFAULT('bydefault'),
+
+        STAGE('stage'),
+
+        NO_STAGE('nostage'),
+
+        EXTERNAL_STAGE('external_stage')
+
+        private final String stringRepresentation
+
+        WeblogicDeploymentStageModes(String stringRepresentation) {
+            this.stringRepresentation = stringRepresentation
+        }
+
+        @Override
+        String toString() {
+            this.stringRepresentation
+        }
+    }
+
+    List<Node> taskNodes = []
+    List<Node> deploymentPoliciesIdsNodes = []
+
+    boolean mustExitOnFailure = false
+    boolean forceStopOnFirstFailure = false
+    boolean deployingOnlyWhenUpdates = false
+    String deployedProjectsDependencies = ''
+
+    /**
+     * Fails the build if the deployment fails. Defaults to {@code false}.
+     *
+     * @param mustExitOnFailure true, false (default)
```

Comment:
remove the `@param` attributes, the API viewer does not display them and we don't use them for the DSL